database/index.js

```diff
@@ -0,0 +1,59 @@
+const mysql = require('mysql');
+const mysqlConfig = require('./config.js');
+
+const connection = mysql.createConnection(mysqlConfig);
+
+function getRandomInt(min, max) {
```

Comment:
Super nit-picky, would maybe name function 'generateRandomInt' just because you aren't 'getting' anything but rather 'creating' something.